template.adoc

```diff
@@ -58,7 +58,9 @@ include::{common-includes}/gitclone.adoc[]
 // they will be building.
 === Try what you'll build
 
+include::{common-includes}/tryintro.adoc[]
 

```

Comment:
In between here, perhaps you want to add a comment for guide writer to add the talk about what user are expected to see. ie, // Describe what user are expected to see

Reply:
Fixed